core/domain/suggestion_jobs_one_off.py

```diff
@@ -179,3 +179,34 @@ def reduce(key, values):
                 no_of_suggestions_migrated)])
         else:
             yield (key, values)
+
+
+class SuggestionLanguageCodeMigrationOneOffJob(
+        jobs.BaseMapReduceOneOffJobManager):
+    """A reusable one-time job that may be used to add the language_code field
+    to suggestions that do not have that field yet. The language_code field
+    allows question and translation suggestions to be queried by language.
+    This job will load all existing suggestions from the data store, update
+    them and immediately store them back into the data store.
+    """
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [suggestion_models.GeneralSuggestionModel]
+
+    @staticmethod
+    def map(item):
+        if item.deleted:
+            return
+
+        # Getting the suggestion from the model creates the corresponding
+        # suggestion domain object. In the init functions for the suggestion
+        # domain classes, the language code field is retrieved and set
+        # automatically.
+        suggestion = suggestion_services.get_suggestion_from_model(item)
+        suggestion_services.update_suggestion(suggestion)
```

Comment:
Either:

(a) do the update only for suggestions which do need an update, or

(b) look into update_last_updated_time=False to ensure that this doesn't update the last-updated timestamps of content-edit suggestion models